### DISCUSSION TITLE
Existing BS Data Editor - things we DON'T like

### DISCUSSION THREAD

**Discussion by amis92** [BSData/phalanx#11] [Ideas]
Please add any and all the *bad* features of the existing BS data editor.

**Comment by cartag** [comment]
Not able to select multiple items at once to copy. 
Cannot Right Click a Category and see Usages like you can with SSEs. 
Cannot type in Publication box to filter results of dropdown
![image](https://user-images.githubusercontent.com/4388344/143323046-c2b79d2e-65e4-4f76-b103-8c8d9a410f92.png)
Cannot type in Profile type box to filter results of dropdown
![image](https://user-images.githubusercontent.com/4388344/143323067-d046fa71-4ecc-4d3c-9968-6a2f7339113c.png)

**Comment by cartag** [comment]
New Info Link defaults to Rule rather than Profile.

Constraint needs to remain clickable and have side menu for Quick Constraints of Min/Max 1 Parent and Roster rather than have them at the bottom of the main data screen for an item. 
![image](https://user-images.githubusercontent.com/4388344/143323496-1c8e5083-0f62-4e67-97cc-051af1f15340.png)

**Comment by cartag** [comment]
The completely unnecessary Comment Box which doesn't do anything
![image](https://user-images.githubusercontent.com/4388344/143324751-04db7aa5-20f6-4f34-856f-8e02bc33bb78.png)

**Reply by jlueck6** [reply]
I think it's useless 99% of the time, esp for the screen space it takes up. Personally, I've used it before to indicate a field I thought was useless was in fact the thing allowing Units to modify one of the other elements' limits. I would vote "hide this by default, but show it if someone actually added a comment via lower-button press". Maybe just axe the field for data editor MVP and relitigate

**Reply by amis92** [reply]
It was meant for developers to add comments explaining complex or unobvious solutions. I don't think it's used a lot. Also, there was an expectation these comments could be shown to users as an Information hover or something, explaining to them some of data author design decisions.

**Reply by cartag** [reply]
Since it seems it's supposed to serve some purpose, maybe move to the bottom of the screen or have it as an item to add like you would add a profile or a rule?

**Comment by cartag** [comment]
It just keeps getting bigger each time I open the program. It may be a bug on my system, but it just keeps. Getting. BIGGER. And then I have to resize the window to keep it less than full screen maximized.

**Comment by cartag** [comment]
That it doesn't actually hide Categories marked as hidden.

**Reply by rahtolsGitHub** [reply]
No search box for things already entered, you have to scroll up and down for ages in larger files.

No "repeat n-1 times". Annoying, when you enter the wargear's cost in it's entry separately and then have an unit have it, not on a per model basis.

 My thoughts, but I'm also no professional Data editor...

**Reply by rahtolsGitHub** [reply]
When entering Characteristics, jump to the next field by pressing TAB.



*edit by Amis92: hidden the image, as it was huge and the description is already quite precise. Also, please post ideas as separate comments, not replies on other comments.*
